docs/docs/schema-customization.md

```diff
@@ -303,37 +303,37 @@ schema.buildObjectType({
++   // controlar comportamento de inferência, Construtores de Tipo do Gatsby recebem um única extensão 
++   // `infer` que aceita um Boolean
 +   infer: false
 + },
 }),
 ```
 
-> Type Builders also exist for Input, Interface and Union types:
-> `buildInputType`, `buildInterfaceType`, and `buildUnionType`.
-> Note that the `createTypes` action also accepts `graphql-js` types directly,
-> but usually either SDL or Type Builders are the better alternatives.
+> Os Construtores de Tipo também existem para os tipos Input, Interface e Union:
+> `buildInputType`, `buildInterfaceType`, e `buildUnionType`.
+> Note que a action `createTypes` também aceita tipos `graphql-js` diretamente,
+> mas geralmente SDL ou Construtores de Tipo são as melhores alternativas.
 
-#### Foreign-key fields
+#### Campos de chave estrangeira
 
-Gatsby's automatic type inference has one trick up its sleeve: for every field
-that ends in `___NODE` it will interpret the field value as an `id` and create a
-foreign-key relation.
+A inferência automática de tipo do Gatsby tem um truque na manga: para cada campo
+que termina em `___NODE` ele interpretará o valor do campo como um `id` e cria uma
+relação de chave estrangeira.
 
-> Note: Before the introduction of the Schema Customization APIs in Gatsby v2.2,
-> there were two mechanisms to create links between node types: a plugin author would use the `___NODE`
-> fieldname convention (for plugins), and a user would define [mappings](/docs/gatsby-config/#mapping-node-types) between fields in their `gatsby-config.js`. Both users and plugin authors can now use the `@link` extension described below.
+> Nota: Antes da introdução das APIs de personalização de Schema no Gatsby v2.2,
+> havia dois mecanismos para criar links entre tipos de node: um autor de plugin usaria a convenção de
+> nome de campo `___NODE` (para plugins), e um usuário definiria [mapeamentos](/docs/gatsby-config/#mapping-node-types) entre campos em `gatsby-config.js`. Ambos, usuários e autores de plugins agora podem usar a extensão `@link` descrita abaixo.
 
-Creating foreign-key relations with the `createTypes` action,
-i.e. without relying on type inference and the `___NODE` field naming
-convention, requires a bit of manual setup.
+Criar relações de chaves estrangeiras com a action `createTypes`,
+ou seja, sem depender da inferência de tipo e da convenção de nomenclatura de campo
+`___NODE`, requer um pouco de configuração manual.
 
-In the example project, the `frontmatter.author` field on
-`MarkdownRemark` nodes to expand the provided field value to a full `AuthorJson` node.
-For this to work, there has to be provided a custom field resolver. (see below for
-more info on `context.nodeModel`)
+No projeto de exemplo, o campo `frontmatter.author` nos nodes
+`MarkdownRemark` para expandir o valor do campo fornecido para um node completo `AuthorJson`.
+Para que isso funcione, é necessário fornecer um resolver de campo personalizado. (veja abaixo para
```

Comment:
```suggestion
Para que isso funcione, é necessário fornecer um _resolver_ de campo personalizado. (veja abaixo para
```

Reply:
Achei interessante deixar como itálico o termo _resolver_ em diversos pontos pois ele pode ser confundido com a palavra "resolver" do português, e caso o usuário queira buscar algo relacionado a este contexto ele terá que utilizar o resolver como termo técnico.